docs/revoguard-hack-2024

```diff
@@ -0,0 +1,97 @@
+# Token Revocation Hackathon: RevoGuard
```

Comment:
Do you want to put this in an "experimental" or "prototype" folder instead?

Reply:
created a `prototype` and moved it there.